test/provisioner/provision.go

```diff
@@ -131,11 +131,20 @@ func (p *CloudAPIAdaptor) Delete(ctx context.Context, cfg *envconf.Config) error
 		return err
 	}
 
-	log.Info("Uninstall CoCo and cloud-api-adaptor")
+	log.Info("Uninstall the cloud-api-adaptor")
 	if err = p.installOverlay.Delete(ctx, cfg); err != nil {
 		return err
 	}
 
+	log.Info("Uninstall CoCo")
```

Comment:
nit: Is "Uninstall CCRuntime CRD" a better log message for this block ?

Reply:
Yeah, that's fair. I struggled with the log messages to find the balance between accuracy and being clear to an end user, so all help and suggestions gratefully accepted!